packages/blueprints/mirror-to-codecommit/README.md

```diff
@@ -0,0 +1,11 @@
+# Mirror to CodeCommit
+
+This blueprint provides a workflow to automatically sync your Amazon CodeCatalyst source repositories to CodeCommit. To use this, add the blueprint to
+your project and specify which branch to sync.
+
+This workflows assumes that codecatalyst is the source of truth. Whenever a commit is pushed to codecatalyst main, it will be force pushed to
```

Comment:
Why can't we make this push to any git repository?

Reply:
Sure I dont see why not

Reply:
Hmmm I actually like this a lot. let me think about to how to properly support this